Dockerfile

```
@@ -0,0 +1,65 @@
+# =======================================
+FROM helsinkitest/node:12-slim as appbase
```

The Dockerfile does not work because the Node is too old:
```
#8 53.64 error msw@0.41.0: The engine "node" is incompatible with this module. Expected version ">=14". Got "12.22.12"
#8 53.65 error Found incompatible module.
```

```suggestion
FROM helsinkitest/node:14-slim as appbase
```